client.go

```diff
@@ -99,6 +103,9 @@ type client struct {
 func serverNameToAddr(server *proto.ServerName) string {
 	return fmt.Sprintf("%s:%d", server.GetHostName(), server.GetPort())
 }
+func addrAndServiceToCache(addr string, srvType ServiceType) string {
```

Comment:
Add an empty space.
addrAndServiceToCache seems not a good name, how about connCacheKey?